examples/fluids/blasius.yaml

```diff
@@ -30,17 +30,17 @@ dm_plex_box_lower: 0,0,0
 dm_plex_box_upper: 4.2e-3,4.2e-3,5.e-5
 dm_plex_dim: 3
 # Faces labeled 1=z- 2=z+ 3=y- 4=y+ 5=x+ 6=x-
-bc_slip_z: 1,2
+bc_symmetry_z: 1,2
 bc_wall: 3
 wall_comps: 1,2,3
 bc_inflow: 6
 bc_outflow: 5,4
 gravity: 0,0,0
 
-stg:
-  use: false
-  inflow_path: "./STGInflow_blasius.dat"
-  mean_only: true
+# stg:
+#   use: false
+#   inflow_path: "./STGInflow_blasius.dat"
+#   mean_only: true
```

Comment:
accidental dead code?

Reply:
It's required to pass tests. I have the code set to warn if there are unused options in test mode (from being bit by that before). Since `stg_use == false`, then the other two options aren't read, so it causes the test to fail on non-empty stdout.

Reply:
Ah. Dead code is usually a 'no-no' in the repo

Reply:
We comment out example options in a few yaml files, mostly for code as a "demonstration" of what can be done, but commented out because it imposes extra library requirements (CGNS for the `ts_monitor_solution` viewer) or aren't really minimum working examples.

Those options don't get tested though, so I'm ok to remove them. I'm curious what @jedbrown thought it, as I think he was the first to put them in.